crates/cxx-qt-build/src/lib.rs

```diff
@@ -255,13 +256,35 @@ fn generate_cxxqt_cpp_files(
     generated_file_paths
 }
 
+/// The export directory, if one was specified through the environment.
+/// Note that this is not namspaced by crate.
+fn export_dir() -> Option<PathBuf> {
```

Comment:
Can be done in the next pull, but splitting things into different modules would be useful so we don't have a massive lib.rs. Eg these are all dirs or info that could go in their own mod.rs